Module/src/PSWordCloud/NewWordCloudCommand.cs

```diff
@@ -809,6 +808,53 @@ protected override void EndProcessing()
 
         #region HelperMethods
 
+        private IEnumerable<string> NormalizeInput(PSObject input)
+        {
+            if (MyInvocation.ExpectingInput)
+            {
+                if (input.BaseObject is string s)
+                {
+                    yield return s;
+                }
+                else
+                {
+                    yield return LanguagePrimitives.ConvertTo<string>(input);
```

Comment:
Don't want to support `,(0, 1) | command`? 😉

Reply:
Nope!

I want `$array | New-WordCloud` and `New-WordCloud -InputObject $array` to behave about the same.

Reply:
Yeah, my example is an `object[][]`, so when it gets to you via the pipeline it'll be an `object[]`, but `ExpectingInput` will be true.  I guess it's technically still handled the same, but a little confusing.  And what if you want to pass an object that is enumerable but you want it treated as a string?  I guess you'd have to convert it yourself 🤷‍♂ 

There's a reason they typically act differently 😛

Reply:
I think it's fair to call that a pretty big edge case 😂

Reply:
Agree to disagree 🙃

Reply:
Hmm. I mean technically it's _less work_ for me not to care if it's piped or not... 🤔

Ya know... why the hell not lol.